src/test/suite/fixtures/recorded/languages/typescript/chuckTypeSoon2.yml

```diff
@@ -0,0 +1,33 @@
+spokenForm: chuck type soon
```

Comment:
Ah this is nice but I actually meant chucking the entire interface, which can also be referred to as "type", just to make sure that wasn't broken here as it has no leading `:`.  But don't remove this test; it's nice too 😊

Reply:
I didn't know that. Test added.

Is just optional leading delimiter. But always good have tests.